node/authorship/src/tests.rs

```diff
@@ -462,7 +462,7 @@ fn queue_remains_intact_if_processing_fails() {
         BlockId::number(0),
         Box::new(time::Instant::now),
         timestamp,
-        1500_u64,
+        2000_u64,
```

Comment:
what does this limit mean? And how we can calculate it using chain block production time limit?

Reply:
This is the actual time limit for block production: 1/2 of a slot duration (3s), which will be later reduced again by 1/3 inside the proposing routine so that we have that 1s for extrinsics processing that is expected by the runtime.

Reply:
did it actually help resolve CI issue? problem occurs on master

Reply:
this actually shouldn't be increased just for passing tests, because its time dependent. separate CI step for only this ignored test, aggregating all of them in just one thread